implementation/contracts/system/FeeRebateToken.sol

```diff
@@ -3,6 +3,14 @@ pragma solidity ^0.5.10;
 import "openzeppelin-solidity/contracts/token/ERC721/ERC721Metadata.sol";
 import "./VendingMachineAuthority.sol";
 
+/// @title  Fee Rebate Token
+/// @notice The Fee Rebate Token (FRT) is a non fungible token (ERC721)
+///         the ID of which corresponds to a given deposit address. 
+///         If the corresponding deposit is still active, ownership of this token
+///         could result in reimbursement of the singer fee payed to open the deposit.
```

Comment:
Probably worth going into a little more detail about when it will and won't result in that, but this is a great start.